e2e/e2etests/e2etests.go

```diff
@@ -323,6 +329,23 @@ var AllE2ETests = []runner.E2ETest{
 		[]runner.ArgDefinition{},
 		TestERC20DepositAndCallRefund,
 	),
+	/*
+	 Solana tests
+	*/
+	runner.NewE2ETest(
+		TestSolanaIntializeGatewayName,
+		"initialize Solana gateway",
+		[]runner.ArgDefinition{},
+		TestSolanaInitializeGateway,
+	),
```

Comment:
Shouldn't it be part of the localnet initialization?

Reply:
trying

Reply:
https://github.com/zeta-chain/node/pull/2465/commits/2743f9dc1b608f81928dc99e067e9f65fcc6d716